.tx/config

```diff
@@ -3,7 +3,7 @@ host = https://www.transifex.com
 
 [friendica.messagespo]
 file_filter = view/lang/<lang>/messages.po
-source_file = util/messages.po
+source_file = view/lang/C/messages.po
```

Comment:
Because I was rewriting the translation documentation by porting over most of `util/README` but its contents were either obsolete or confusing, so I ended up deleting it. Also it didn't make sense that the master PO file wasn't in the same directory as all other translation files. And we didn't use `util/strings.php` so I deleted it as well. From then it wasn't a big step to move `testargs.php` to `bin/` where it should always have been, and the rest of the files in `mods/`.